charts/cinder/templates/deployment-volume.yaml

```diff
@@ -286,6 +288,8 @@ spec:
 {{- dict "enabled" $envAll.Values.manifests.certificates "name" $envAll.Values.endpoints.oslo_messaging.auth.admin.secret.tls.internal "path" "/etc/rabbitmq/certs" | include "helm-toolkit.snippets.tls_volume_mount" | indent 12 }}
 {{ if $mounts_cinder_volume.volumeMounts }}{{ toYaml $mounts_cinder_volume.volumeMounts | indent 12 }}{{ end }}
       volumes:
+        - name: cinder-tmp
+          emptyDir: {}
```

Comment:
considering the possibility for non-malicious abuse here, it would be nice to set a [sizeLimit](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#emptydirvolumesource-v1-core) to prevent a single user from completely filling up the disk area where this is allocated from.

Reply:
huuum that could be a good approach, but im thinking of the limitations that could cause. 

In one way, I completely agree that having this conversion locally on a controller and having no control over the sizes is a dangerous thing.... thats WHY we are disabling this upstream. 

On the other hand, it's a limitation for users to be always using a raw images. Thats difficult to control unless we explicitly tell them the risks and create a note for ALL the users to be only using RAW images